src/components/SectionHowItWorks/SectionHowItWorks.js

```diff
@@ -47,12 +47,6 @@ const SectionHowItWorks = props => {
           </p>
         </div>
       </div>
-
-      <div className={css.createListingLink}>
-        <NamedLink name="NewListingPage">
-          <FormattedMessage id="SectionHowItWorks.createListingLink" />
-        </NamedLink>
-      </div>
```

Comment:
We would need to know if the user already has a listing or not so that we don't show a link to create a listing page when the user already has a listing. We don't currently have that information on LandingPage so I think the easiest solution is to remove this link.

Reply:
Let's keep this - but as you suggested IRL, we shouldn't show this to user's who have already one listing. And let's not render it on SSR. (You should copy the isJustAuthenticatedOrMounted functionality from TopbarDesktop)
You could possibly use `OwnListingLink` here.

https://github.com/sharetribe/flex-template-web/blob/master/src/containers/LandingPage/LandingPage.js#L99
There you need to add:
```js
const { currentUserListing } = state.user;
```
(or something like that.)